stdlib/database/h2/src/test/java/org/ballerinalang/stdlib/utils/SQLDBUtils.java

```diff
@@ -0,0 +1,245 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.stdlib.utils;
+
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Util class for SQL DB Tests.
+ *
+ * @since 0.990.3
+ */
+public class SQLDBUtils {
+
+    public static final String DB_DIRECTORY = "./target/tempdb/";
+    public static final String DB_DIRECTORY_H2_1 = "./target/H2_1/";
+    public static final String DB_DIRECTORY_H2_2 = "./target/H2_2/";
+    private static final Logger log = LoggerFactory.getLogger(SQLDBUtils.class);
+
+    /**
+     * Create H2 DB with the given name and initialize with given SQL file.
+     *
+     * @param dbDirectory Name of the DB directory.
+     * @param dbName      Name of the DB instance.
+     * @param sqlFile     SQL statements for initialization.
+     */
+    public static void initH2Database(String dbDirectory, String dbName, String sqlFile) {
+        String jdbcURL = "jdbc:h2:file:" + dbDirectory + dbName;
+        initDatabase(jdbcURL, "sa", "", sqlFile);
+    }
+
+    /**
+     * Create a DB and initialize with given SQL file.
+     *
+     * @param jdbcURL  JDBC URL
+     * @param username Username for the DB
+     * @param password Password to connect to the DB
+     * @param sqlFile  SQL statements for initialization.
+     */
+    public static void initDatabase(String jdbcURL, String username, String password, String sqlFile) {
+        try (Connection connection = DriverManager.getConnection(jdbcURL, username, password);
+             Statement st = connection.createStatement()) {
+            String sql = readFileToString(sqlFile);
+            String[] sqlQuery = sql.trim().split("/");
+            for (String query : sqlQuery) {
+                st.executeUpdate(query.trim());
+            }
+            if (!connection.getAutoCommit()) {
+                connection.commit();
+            }
+        } catch (SQLException e) {
+            log.error("Error while initializing database: ", e);
+        }
+    }
+
+    /**
+     * Delete the given directory along with all files and sub directories.
+     *
+     * @param directory Directory to delete.
+     */
+    public static boolean deleteDirectory(File directory) {
+        if (!directory.isDirectory()) {
+            return directory.delete();
+        }
+        for (File f : directory.listFiles()) {
```

Comment:
Shall we use something like this to clear directory?
`Files.walk(rootDirectory.toPath(), FileVisitOption.FOLLOW_LINKS)
                        .map(Path::toFile)
                        .forEach(File::delete);`